styles.css

```diff
@@ -0,0 +1,154 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+body {
+    font-size: 120%;
+    font-family: 'Vesper Libre', serif;
+    line-height: 1.4;
+    background-color: #F1D7BC;
+    color: #F1D7BC;;
+}
+h2 {
+    padding:  1rem 0 1rem 0;
+}
+nav {
+    background-color: #3F0900;
+}
+nav ul {
+    list-style: none;
+    display: flex;
+    padding: 0rem 0rem 0rem 4rem;
+}
+nav a {
+    color: #F1D7BC;;
+    text-decoration: none;
+    display: block;
+    padding: 1rem;
+}
+footer {
+    background-color: #3F0900;
+    font-size: smaller;

```

Comment:
es ist besser mit werten zu arbeiten als mit diesen Angaben. smaller, bigger kann man in den verschiedenen Browsern schwer kontrollieren